src/BlitHelper.cpp

```diff
@@ -163,6 +163,8 @@ namespace D3D12TranslationLayer
     void BlitHelper::Blit(Resource *pSrc, UINT *pSrcSubresourceIndices, UINT numSrcSubresources, const RECT& srcRect, Resource *pDst, UINT *pDstSubresourceIndices, UINT numDstSubresources, const RECT& dstRect, bool bEnableAlpha, bool bSwapRBChannels)
     {
         const D3D12_RESOURCE_DESC &dstDesc = pDst->GetUnderlyingResource()->GetDesc();
+        std::vector<UINT> nonMsaaSrcSubresourceIndices( pSrcSubresourceIndices, pSrcSubresourceIndices + numSrcSubresources );

```

Comment:
This is going to be 1-3 values, with 1 being the most common case. Let's use a stack allocation for this, since 4 bytes for a heap allocation is a bit overkill.

Reply:
And actually, thinking further, we could assert that it's only 1 because you can't have MSAA YUV resources. It *might* be possible to have 2, because of depth/stencil, but I don't think we support blits that aren't basic copies/resolves on depth/stencil.